src/App.js

```diff
@@ -12,32 +12,53 @@ import {
 } from "react-router-dom";
 import ProductDetail from "pages/ProductDetail/ProductDetail";
 import SearchResults from "pages/SearchResults/SearchResults";
+import CartContext from "state/CartContext";
+import { useState } from "react";
+import ShoppingCart from "pages/ShoppingCart/ShoppingCart";
+import Checkout from "pages/Checkout/Checkout";
 
 function App() {
+  const [products, setProducts] = useState([]);
+  const [totalProducts, setTotalProducts] = useState(0);
+
+  // useEffect(() => {
+  //   console.log("cart products", products);
+  //   console.log("total prods", totalProducts);
+  // }, [products]);
```

Comment:
We should remove this if it's not going to be used

Reply:
Done